python/instrumentation/openinference-instrumentation-openai/src/openinference/instrumentation/openai/_utils.py

```diff
@@ -24,7 +25,10 @@
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
-_OPENAI_VERSION = tuple(map(int, version("openai").split(".")[:3]))
+
+@lru_cache
+def _openai_version() -> Tuple[int, int, int]:
+    return cast(Tuple[int, int, int], tuple(map(int, version("openai").split(".")[:3])))
```

Comment:
```suggestion
@lru_cache
def _get_openai_version() -> Tuple[int, int, int]:
    return cast(Tuple[int, int, int], tuple(map(int, version("openai").split(".")[:3])))
```
a bit more clear that it's a function rather than a constant